### PR TITLE
fix: added validation for doctor experience input.

### LINKED ISSUE #7407: [BUG] No validation when doctor experience input is less than 0.
**Describe the Bug:**
In users' personal experience, while updating the user profile, there is validation on Age, etc., but there seems to be no validation on Experience when the experience is less than 0.

**To Reproduce:**

1. Go to the User Profile.
2. Click on Edit User Profile.
3. Set experience to -35 (for example).
4. It will be saved with no validation.

**Expected Behavior:**
There must be validation to verify if the experience is at least 0 and shouldn't be less than that.

**Screenshots / Videos:**

[Link to screenshot or video](https://github.com/coronasafe/care_fe/assets/78961432/f9989aeb-164f-4fbf-95f6-5dcee05a4caf)

**Desktop:**

- OS: Ubuntu 22.04.3 LTS
- Browser: Microsoft Edge
- Version: 120.0.2210.133 (Official build) (64-bit)

### PR DESCRIPTION
**Pull request by Spiral-Memory**
## Proposed Changes

- Fixes #7407 
- Added input validation when experience is less than 0
- Accordingly adjusted the message of the error message

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

https://github.com/coronasafe/care_fe/assets/78961432/b5a18e0e-702c-476e-9ce1-c82d83eddd44

### CODE REVIEW

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Users/UserProfile.tsx [hunk 1]
```diff
@@ -249,10 +249,12 @@ export default function UserProfile() {
             errors[field] = "Field is required";
             invalidForm = true;
           } else if (
-            states.form.user_type === "Doctor" &&
-            Number(states.form.doctor_experience_commenced_on) > 100
+            (states.form.user_type === "Doctor" &&
+              Number(states.form.doctor_experience_commenced_on) > 100) ||
```
Comment (Ashesh3) [inline src/Components/Users/UserProfile.tsx]: > "Doctor experience should be at least 0 years and less than 100 years."

According to the error, shouldn't the condition here be `>= 100` ? (If equal or more than 100, tell them that it needs to be LESS than 100)

```suggestion
              Number(states.form.doctor_experience_commenced_on) >= 100) ||
```

**Review by Ashesh3 — CHANGES_REQUESTED** [review]

File: src/Components/Users/UserProfile.tsx (hunk 1 above)
Comment (Spiral-Memory) [reply src/Components/Users/UserProfile.tsx]: Yes, that can be added. However it was already there like this only, so i only added the < 0 condition and didn't touched this condition.

Sure, let me change that !

Comment (Spiral-Memory) [reply src/Components/Users/UserProfile.tsx]: Done @Ashesh3

**Review by Ashesh3 — APPROVED** [review]

**Comment by Spiral-Memory** [comment]
Thank you so much for approving the changes. Sorry if I'm asking something irrelevant; I'm new to this repository. May I know how long testing usually takes? @Ashesh3  @rithviknishad

**Comment by Ashesh3** [comment]
> Thank you so much for approving the changes. Sorry if I'm asking something irrelevant; I'm new to this repository. May I know how long testing usually takes? @Ashesh3 @rithviknishad

The PRs approved are usually tested within the upcoming week by @nihal467

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**